beets/util/artresizer.py

```diff
@@ -92,11 +93,13 @@ def im_resize(maxwidth, path_in, path_out=None):
     # than the given width while maintaining the aspect ratio
     # with regards to the height.
     try:
-        util.command_output([
-            'convert', util.syspath(path_in, prefix=False),
-            '-resize', '{0}x>'.format(maxwidth),
-            util.syspath(path_out, prefix=False),
-        ])
+        cmds = (['magick'], ['convert'])
```

Comment:
Consider hoisting this out to a module-level constant, named something like `CONVERT_CMDS`.

Reply:
Went with your second suggestion here